### ISSUE TITLE
clang version has rc suffix in 18.1.0 release

### ISSUE THREAD

**Issue by stylesen** [ClangBuiltLinux/linux#2005]
In [Prebuilt LLVM 18.1.0 uploaded](https://mirrors.edge.kernel.org/pub/tools/llvm/), the clang version has `rc` suffix.

```
# clang --version
ClangBuiltLinux clang version 18.1.0rc (https://github.com/llvm/llvm-project.git 461274b81d8641eab64d494accddc81d7db8a09e)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/local/llvm-18.1.0-x86_64/bin
```

The `rc` suffix should be removed in LLVM 18.1.0 release.

**Comment by nathanchance** [comment]
This is an upstream issue: https://github.com/llvm/llvm-project/issues/84168

**Comment by beakthoven** [comment]
This issue has been fixed in [llvmorg-18.1.1](https://github.com/llvm/llvm-project/releases/tag/llvmorg-18.1.1)
